wled00/data/index.js

```diff
@@ -1697,7 +1698,7 @@ function toggleLiveview()
 	}
 
 	gId(lvID).style.display = (isLv) ? "block":"none";
-	gId(lvID).src = (isLv) ? getURL("/" + lvID + ((wsOn) ? "?ws":"")):"about:blank";
+	gId(lvID).src = (isLv) ? getURL("/" + lvID):"about:blank";
```

Comment:
Why did you remove this? liveview uses the `?ws` parameter to determine if it can use WS, without it it will always fallback to json.

Reply:
I am sorry. I missed the "?" in the string.